guides/source/active_record_connection_management.md

```diff
@@ -0,0 +1,107 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON http://guides.rubyonrails.org.**
+
+Active Record Connection Management
+===================================
+
+In general, Active Record will automagically manage database connections in an
+effecient manner. See [Configuring a Database](/configuring.html#configuring-a-database) and [Database pooling](configuring.html#database-pooling) for information about the basics of optimally configuring your application to use available resources.
+
+Sometimes, you will need to manage Active Record connections and/or connecton
+pools yourself. These situations are:
+
+* Configuring a multi-process web server
+* Spawning processes
+
+In previous version of rails, you needed to think about managing connections when
+spawning threads, but this is no longer the case.
```

Comment:
@arthurnn when i said "when spawning threads", i mean when creating new threads in your application code. regarding that, my statement is correct.

but it seems you thought i meant "you need to consider the number of threads your server is running when you decide how many connections to put in the pool".

if you have a suggestion how to clarify that it is the former and not the latter, let me know!

(or if i'm not understanding your feedback, let me know that too :-D)